vimeo-networking/src/main/java/com/vimeo/networking/model/iap/Product.java

```diff
@@ -0,0 +1,110 @@
+package com.vimeo.networking.model.iap;
+
+import com.google.gson.annotations.SerializedName;
+import com.vimeo.networking.model.Metadata;
+import com.vimeo.stag.UseStag;
+
+import org.jetbrains.annotations.Nullable;
+
+import java.io.Serializable;
+
+/**
+ * Represents a Product that can be purchased, such as a Vimeo subscription.
+ *
+ * Created by brentwatson on 10/30/17.
+ */
+@SuppressWarnings("unused")
+@UseStag
+public class Product implements Serializable {
+
+    private static final long serialVersionUID = 27901675080110352L;
+
+    @Nullable
+    @SerializedName("product_id")
+    protected String mProductId;
```

Comment:
Why protected?

Reply:
Since `Product` is a new object I'll update to use `private` fields. 👍.  I originally them as `private` this but other models seem to use protected fields (I think a legacy stag limitation?) so I had changed to be consistent.  I see a couple other models do have private fields though, so I guess a mix is OK for now.